src/lib/main/main-draft.js

```diff
@@ -0,0 +1,23 @@
+async function pullRequestOpened(context) {
+  context.github.issues.addLabels(
+    context.issue({
+      labels: ["WIP :construction_worker_man:"]
+    })
+  );
+}
+
+async function pullRequestReopened(context) {
+  // nothing to do
```

Comment:
remove `return` and `// nothing to do` line and log warning with text `no pull request reopened for draft`